source/index.html.md

```diff
@@ -154,6 +154,7 @@ error categories, the last two digits define specific errors.
 * 0209: Two Factor authentication required
 * 0210: Device authentication required
 * 0211: Device authentication failed (Invalid IP or expired Token)
+* 0212: IP address is not allowed to access this resource
```

Comment:
0212 is already used by Quoterbot :/

Reply:
Thanks @thenewversion please do you know what message it returns so i document while i take the next available code `0213` ?

Reply:
It simply returns access denied but is very specific for Quoterbot so it should not really be added to that list unless marked as `reserved for internal use`

Reply:
Okay thanks